src/deploy-tes-on-azure/Deployer.cs

```diff
@@ -1105,11 +1104,13 @@ private async Task EnableWorkloadIdentity(ManagedCluster aksCluster, IIdentity m
         {
             settings ??= new();
             var defaults = GetDefaultValues(new[] { "env-00-tes-version.txt", "env-01-account-names.txt", "env-02-internal-images.txt", "env-04-settings.txt" });
+            var currentTime = DateTime.UtcNow;
 
             // We always overwrite the CoA version
             UpdateSetting(settings, defaults, "TesOnAzureVersion", default(string), ignoreDefaults: false);
             UpdateSetting(settings, defaults, "ResourceGroupName", configuration.ResourceGroupName, ignoreDefaults: false);
             UpdateSetting(settings, defaults, "RegionName", configuration.RegionName, ignoreDefaults: false);
+            UpdateSetting(settings, defaults, "DeploymentUpdated", currentTime.ToString("O"), ignoreDefaults: false);
```

Comment:
This line should actually set `ignoreDefaults: true`

Reply:
Yes, I know the lines above it have that set to false. There's a rhyme to the madness. This value is different than those others, only sharing the fact that it needs to be rewritten every time.

Reply:
I'm wrong (it's been a very long time since I wrote that code). `false` is correct.